exporter/elasticsearchexporter/README.md

```diff
@@ -337,4 +337,12 @@ Otherwise, it is mapped to an empty string ("").
 
 #### `@timestamp`
 
-In case the record contains `timestamp`, this value is used. Otherwise, the `observed timestamp` is used.
\ No newline at end of file
+In case the record contains `timestamp`, this value is used. Otherwise, the `observed timestamp` is used.
+
+## Known issues
+
+### version_conflict_engine_exception
+
+When sending high traffic of metrics to a TSDB metrics data stream, e.g. using OTel mapping mode to a 8.16 Elasticsearch, it is possible to get error logs "failed to index document" with `error.type` "version_conflict_engine_exception" and `error.reason` containing "version conflict, document already exists". It is due to Elasticsearch grouping metrics using `@timestamp` in milliseconds precision as opposed to nanoseconds in elasticsearchexporter. This will be fixed in a future version of Elasticsearch.
```

Comment:
(This paragraph being a single line makes it a bit hard to add review comments on specific parts)

> using OTel mapping mode to a 8.16 Elasticsearch

Does this error happen with Elasticsearch 8.16? Version 8.16 It is not released yet, right?

Reply:
It is not released, but no fix is planned for 8.16, and it will be a known issue.